raiden/network/transport/matrix/transport.py

```diff
@@ -1338,33 +1338,61 @@ def _send_with_retry(self, queue: MessagesQueue) -> None:
 
     def _send_raw(self, receiver_address: Address, data: str) -> None:
         assert self._raiden_service is not None, "_raiden_service not set"
-        room = self._get_room_for_address(receiver_address, require_online_peer=True)
 
+        # Try sending using webRTC
+        if self._web_rtc_manager.has_ready_channel(receiver_address):
+            self.log.debug(
+                "Send raw using webRTC message",
+                receiver=to_checksum_address(receiver_address),
+                data=data.replace("\n", "\\n"),
+            )
+            rtc_partner = self._web_rtc_manager.address_to_rtc_partners[receiver_address]
+            send_rtc_message(rtc_partner, data, self._raiden_service.address)
+            return
+
+        # Try sending using toDevice message
+        capabilities = self._address_mgr.get_address_capabilities(receiver_address)
+        to_device_key = Capabilities.TODEVICE.value
+        if to_device_key in capabilities and capabilities[to_device_key]:
+            self.log.debug(
+                "Send raw using toDevice message",
+                receiver=to_checksum_address(receiver_address),
+                data=data.replace("\n", "\\n"),
+            )
+            online_userids = {
+                user_id
+                for user_id in self._address_mgr.get_userids_for_address(receiver_address)
+                if self._address_mgr.get_userid_presence(user_id) in USER_PRESENCE_REACHABLE_STATES
```

Comment:
Maybe we should skip this to lower the change of the presence bug biting us.

Reply:
@ulope @fredo With this the SP integration test fails. Is it possible that we miss the presence updates because we don't open rooms any more?

Reply:
Maybe this `set` should be before the `if` and be skipped if there's no online user? Although I suppose there should be some early check for this case

Reply:
There is an check beforehand for online precence of the peer